infrastructure/cdn-in-a-box/README.md

```diff
@@ -182,3 +182,33 @@ Try disabling SELinux or setting it to 'permissive'. SELinux hates letting conta
 ### Traffic Vault container exits with cp /usr/local/share/ca-certificates cp: missing destination
 
 Bring all components down, remove the `traffic_ops/ca` directory, and delete the volumes with `docker volume prune`. This will force the regeneration of the certificates.
+
+## Notes for Apple M1 Pro Configuration
+
+### Code edits in the cdn-in-a-box directory
+
+Re/installed docker from the command line to use user=[your username] flag, and update the dockerfiles in the code for both enroller and traffic_ops in the cdn-in-a-box folder with the correct architecture for the M1. Link to install info for docker: https://docs.docker.com/desktop/install/mac-install/  and use Install from command line. 
```

Comment:
For the best UX for users that need assistive technologies, links should have descriptive content instead of raw URLs

```markdown
[Like this](https://likethis.com)
```

Reply:
sounds good! (: